data/yam/agama/auto/sles_sap_default_ssh_pub_key.jsonnet

```diff
@@ -0,0 +1,42 @@
+{
```

Comment:
we don't need to copy/paste this file, we could just edit the existing one. Both architecture will have configure the authentication in this way even if it is only required for power.